packages/perl-debug-adapter/package.yaml

```
@@ -0,0 +1,20 @@
+---
+name: perl-debug-adapter
+description: Perl debugger
```

Figured this might be bit more descriptive 😆 
```suggestion
description: Perl debugger that invokes perl -d.
```